.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test build
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test-deploy:
+    name: Test build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Test build website
+        run: yarn build
```

Comment:
Is this how we run middletown? I think it's the build task of docusaurus isn't it?

I'd recommend using these steps:
```yaml
    steps:
      - uses: actions/checkout@v3
      - uses: ruby/setup-ruby@v1
        with:
          bundler-cache: true
      - name: Test build website
        run: bin/bridgetown build
```

Reply:
I didn't give it much of a thought to be honest, what would I do without you 👀